.github/workflows/test-all-commits.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '14'
```

Comment:
Apparently there's a bug on `node` 16 that caused tests not to run, the job failed in the preparation steps. Downgrading to use an earlier version of node solved the issue.

Reply:
Can you create an issue to track this? Would be good to understand this better